### DISCUSSION TITLE
2.0.25

### DISCUSSION THREAD

**Discussion by agronick** [agronick/aa-torque#111] [Announcements]
## What's Changed
* Alarm feature  by @agronick in https://github.com/agronick/aa-torque/pull/109
* New translations by @agronick in https://github.com/agronick/aa-torque/pull/110

**Full Changelog**: https://github.com/agronick/aa-torque/compare/2.0.23...2.0.25

<hr /><em>This discussion was created from the release <a href='https://github.com/agronick/aa-torque/releases/tag/2.0.25'>2.0.25</a>.</em>

**Comment by agronick** [comment]
Give your feedback for the alarm feature and let me know if it is working correctly.

**Reply by Vince56190** [reply]
Hi, it work very well for me  
![5668011114179491229 (1)](https://github.com/agronick/aa-torque/assets/147246549/39f0b7ee-c104-4e9e-b512-b551b0fec1d4)

**Comment by grahamdee63** [comment]
AATorque not available in Android Auto using a wired connection.

Tried :
Clearing AA cache
Phone Reboot
No errors during install, It all went as demonstrated. 

AATorque V2.0.25
Torque V1.12.101
Android Auto V11.1.640234
Android V13
OPPO Find X3 Lite CPH2145

Any ideas?
